src/torchcodec/_frame.py

```diff
@@ -38,6 +38,12 @@ class Frame(Iterable):
     duration_seconds: float
     """The duration of the frame, in seconds (float)."""
 
+    def __post_init__(self):
```

Comment:
Can you describe in a comment why this dunder method was added? (for python newbies)